contents/docs/deployment.mdx

```diff
@@ -438,6 +438,70 @@ Explaining the arguments above --
 - `ZERO_AUTH_SECRET` - The secret to create and verify JWTs. This is the same secret that was used when deploying zero-cache to fly.
 - `VITE_PUBLIC_SERVER` - The URL the frontend will call to talk to the zero-cache server. This is the URL of the fly app.
 
+## Guide: Single-Node on  Sevalla.com
+
+[Sevalla.com](https://sevalla.com) is a modern Platform as a Service (PaaS) that can be used to deploy `zero-cache` in a single-node configuration, similar to Fly.io.
+
+This guide will walk you through deploying `zero-cache` to Sevalla using its web dashboard.
+
+### Setup
+
+First, create an account on [Sevalla.com](https://sevalla.com). You will also need an upstream Postgres database. You can use any hosted Postgres provider like Supabase or Neon. Ensure you have the connection string handy.
+
+### 1. Create a New Application
+
+1.  From the Sevalla dashboard, create a new Application.
+2.  Choose **Deploy from a public Docker image**.
+3.  For the **Image URL**, enter `rocicorp/zero`. You can append a specific version tag like `rocicorp/zero:0.22.2025080201`, or use `latest` to get the newest version.
+4.  Click **Continue**.
+
+### 2. Configure Your Application
+
+On the configuration screen, you will set up the necessary resources, networking, and environment variables.
+
+#### Resources
+
+Under the **Resources** section, it is critical to allocate enough memory. `zero-cache` can be memory-intensive during startup.
+
+-   **RAM**: Set to at least **2 GB**.
+-   **vCPU**: Set to at least **1 vCPU**.
```

Comment:
```suggestion
-   **vCPU**: Set to at least **0.5 vCPU**.
```
By the way, a 0.5 can also be sufficient, like an M1 pod size, but obviously, the bigger the better. 😇